exec_test.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+
+#Make it executable by default (might not be needed)
+chmod +x exec_test.sh
+
+function argsHelp()
+{
+    echo -e "\n *** Usage help ***"
+    echo -e "\n $0 [-t path/to/testcase.py] \n"
+    exit 1
+}
+while getopts ":t:" flag
+do
+    case "$flag" in 
+        t) testcase=${OPTARG} ;;
+        :) argsHelp ;;
+        \?) argsHelp ;;
+    esac
+done
+
+#Check if t param is missing
+if [ -z "$testcase" ]
+then
+    echo -e "\n ### Command line param missing ! ###"
+    argsHelp
+fi
+
+#Build the mta docker image from dockerfile
+docker build dockerfiles/docker_fedora31/
+
+#Run the mta container on port 8080 in detached mode
+if [[ $(docker ps | grep windup_mta) = *windup_mta* ]]
+then
+    echo -e "\n MTA web console container is already running !"
+else
+    if [[ $(docker ps -a | grep windup_mta) = *windup_mta* ]]
+    then
+        echo -e "\n Starting the existing MTA web console docker container !"
+        docker start windup_mta
+    else
+        echo -e "\n Starting new MTA web console container !"
+        docker run -d -p 8080:8080 --name windup_mta -it mta
```

Comment:
I am getting error here that the image does not exist .
While building we need to specify image name I think